src/components/molecules/SpaceEditForm/SpaceEditForm.scss

```diff
@@ -24,6 +24,16 @@
     padding: $spacing--xl;
   }
 
+  &__input {
```

Comment:
Are the selectors under this `&__input` even used anywhere?

Reply:
removed ef337b8